### PR TITLE
Oceanic : revert to simpler byte-driven serial reader thread [After m…

### PR DESCRIPTION
**Pull request by kgoderis**
…oving to a linux based OH server, suddenly the stream of characters received from the Oceanic device contains 0x00 (‘\0’) bytes. I am not sure whether this is due to the serial drivers on Linux, the JVM used (openJDK) or something else] 

The PR was checked by the Static Code Analysis Tool

Signed-off-by: Karel Goderis <karel.goderis@me.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.oceanic/build.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ output.. = target/classes
 bin.includes = META-INF/,\
                .,\
                OSGI-INF/,\
-               ESH-INF/
\ No newline at end of file
+               ESH-INF/
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.oceanic/build.properties]: could you revert the change in this file?

Comment (kgoderis) [inline addons/binding/org.openhab.binding.oceanic/build.properties]: Stupid as it sounds, I have no clue how one can add a newline in Eclipse.

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.oceanic/build.properties]: The "return" key does not work...?
You could actually simply remove this file from your changeset, that would be the simplest solution!

Comment (kgoderis) [inline addons/binding/org.openhab.binding.oceanic/build.properties]: No, it doesn't. But the SCATool tells me that a new line is missing. Will make the quick change

**Comment by kgoderis** [comment]
@kaikreuzer done.

**Comment by kaikreuzer** [comment]
Thanks!
